spec/helpers/webmock_helpers.rb

```diff
@@ -85,6 +85,24 @@ def contact_find_stub(endpoint, id, success = true)
     end
   end
 
+  def contacts_find_query_stub(endpoint, query, has_items = true)
+    if has_items
+      stub_request(
+        :get,
+        "#{endpoint}/private/api/v2/json/contacts/list?query=#{query}")
+        .to_return(
+          body: File.read('./spec/fixtures/contact_find_query.json'),
+          headers: { 'Content-Type' => 'application/json' },
+          status: 200
+        )
+    else
+      stub_request(
+        :get,
+        "#{endpoint}/private/api/v2/json/contacts/list?query=#{query}")
+        .to_return(body: { 'response' => { 'contacts' => [] } }.to_json, status: 200)
```

Comment:
Metrics/LineLength: Line is too long. [85/80]<br>Style/DotPosition: Place the . on the previous line, together with the method call receiver.